site/src/docs/patterns/cookies/basics.mdx

```diff
@@ -0,0 +1,114 @@
+| **Third-party services**      | Cookies allow external services (such as social media platforms) to operate on this website.                                   | Not considered necessary unless cookies are used only after opening the third-party service (e.g. a chat window).                                                                                                                                                                            | Cookies used by embedded media such as videos or podcasts                                    |
+| **Content presentation**      | Cookies are used to display various types of content to the user.                                                              | Can be considered necessary if the cookie is strictly related to presenting the content and not gathering data.                                                                                                                                                                              | Monitoring how the user is using specific content                                            |
+| **Location**                  | Location cookies can be used to utilise location information (GPS) received from the user’s device.                            | Never necessary.                                                                                                                                                                                                                                                                             | Accurate GPS location                                                                        |
+| **Load balancing**            | Load-balancing cookies ensure that the service loads and works quickly and efficiently.                                        | Always necessary (as they ensure that the service is operating smoothly).                                                                                                                                                                                                                    | A client identifying hash to enable better routing                                                                                            |
+[Table 3:Cookie categories]|
+
+#### Can I use a custom category?
+
+If your cookies do not fit into any of the pre-defined categories, you are allowed to use custom categories.
+
+#### We only have a handful of cookies. Do we still have to categorize them?
+
+Yes, cookies always have to be put into a category. If you have a small number of cookies you can consider grouping them into a single category. Also, if most of the cookies are related to a single third-party service, you can group them per service (E.g. "Service: YouTube").
+
+### Prepare your CookieConsent component
```

Comment:
We refer to React's CookieConsent component here? Technically, we currently do not have a component named CookieConsent, which might be little misleading. Instead, we currently expose these two components:

1. `CookieModal`
2. `CookiePage`

Im not sure if we need to be so specific on things. But usually when we refer to a component in the documentation, that same name can be found in the React code also.

Reply:
Added CookieModal/CookiePage to the header